feature-libs/product-configurator/rulebased/cpq/cpq-configurator-overview-normalizer.ts

```diff
@@ -78,12 +78,15 @@ export class CpqConfiguratorOverviewNormalizer
     const ovValues: Configurator.AttributeOverview[] = [];
     switch (attr.displayAs) {
       case Cpq.DisplayAs.INPUT:
-        if (attr.userInput && attr.userInput.length > 0) {
-          ovValues.push(this.extractOvValueUserInput(attr, currency));
+        if (attr?.dataType === Cpq.DataType.INPUT_STRING) {
+          if (attr.userInput && attr.userInput.length > 0) {
+            ovValues.push(this.extractOvValueUserInput(attr, currency));
+          }
+        } else {
+          ovValues.push({ attribute: undefined, value: 'NOT_IMPLEMENTED' });
```

Comment:
this does not seem to be related to 'move value from input field'. did you fix multiple issues with this PR?

Reply:
Yes. Several issues (directly or indirectly related to handling of input fields) are fixed with this pull request. All adjustments are mentioned in the pull request comment.